offices/_single.html

```diff
@@ -106,23 +110,21 @@ <h2 class="h3">{{ resource.title | safe }}</h2>
     </section>
     {% endif %}
 
-    {% if vars.sub_pages %}
+    {% for page in vars.sub_pages if page.show_in_office == "on" %}
+    {% if loop.first %}
```

Comment:
Instead of putting this in the loop, could you move it outside but set it conditionally with `{% if vars.sub_pages | length > 0 %}`